base_tier_validation/models/tier_review.py

```diff
@@ -7,8 +7,9 @@
 class TierReview(models.Model):
     _name = "tier.review"
     _description = "Tier Review"
+    _order = "sequence"
 
-    name = fields.Char(related="definition_id.name", readonly=True)
+    name = fields.Char(compute="_compute_definition_data", store=True, readonly=False)
```

Comment:
kittiu: I need to use compuate store=True, readonly=False of v13, because the extra escalated tier will not have definition. But still need these fields value.